### PR TITLE
logic and props  added

### PR DESCRIPTION
**Pull request by bharti2022**

### CODE REVIEW

File: src/components/atoms/TextField/index.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { TextField } from "@material-ui/core";
+
+const TextField = ({placeholder,onChange,startIcon,endIcon}) => {
+  return <div>TextField</div>;
```
Comment (SravaniKusuma) [inline src/components/atoms/TextField/index.tsx:5]: use mui textfield

File: src/components/molecules/Banner/index.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Banner = () => {
+  return <div>Banner</div>;
```
Comment (SravaniKusuma) [inline src/components/molecules/Banner/index.tsx:4]: add the content for banner, what we will be displaying like image or typography

File: src/components/molecules/ExploreDropDown/ExploreDropDown.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const ExploreDropDown = () => {
+  return (
+    <div>ExploreDropDown</div>
```
Comment (SravaniKusuma) [inline src/components/molecules/ExploreDropDown/ExploreDropDown.tsx:5]: add the tabs and the items you will be displaying in explore dropdown

File: src/components/molecules/Modal/index.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+const Modal = (open, component) => {
+  const [open, isOpen] = useState(open);
+  const handleClick = () => {
+    isOpen(false);
+  };
+  return (
+    <>
+      {open && (
+        <div><Image src="close.png" onClick={handleClick}/>
+          <component />
```
Comment (SravaniKusuma) [inline src/components/molecules/Modal/index.tsx:12]: what exactly the component mean here?

File: src/components/organisms/AutoCompleteSearch/index.tsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const index = (placeholder,startIcon) => {
+  return (
+    <div>AutoCompletewithlistof book</div>
```
Comment (SravaniKusuma) [inline src/components/organisms/AutoCompleteSearch/index.tsx:5]: here as well update this component with proper markup

File: src/components/organisms/Header/index.tsx [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+                                    >
+                                      {suggestion.name +
+                                        " by " +
+                                        suggestion.author}
+                                    </div>
+                                  </>
+                                );
+                              })}
+                          </>
+                        );
```
Comment (SravaniKusuma) [inline src/components/organisms/Header/index.tsx:123]: can you refactor this into a separate component something like SearchResults component

**Review by SravaniKusuma — APPROVED** [review]

**Comment by bharti2022** [comment]
made all changes mentioned in comments

**Comment by bharti2022** [comment]
changes done

**Pull request merged**